assets/src/edit-story/utils/useIntersectionEffect.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+const IntersectionObserver = require('intersection-observer-polyfill/dist/IntersectionObserver');
```

Comment:
Wouldn't `import IntersectionObserver from 'intersection-observer-polyfill';` work?

Reply:
It was changed to fix travis tests it seems - https://github.com/google/web-stories-wp/pull/562/commits/ae012fe31aa7984db6acc3ba812dacf97b78f331

Reply:
Nope, it's up to library author. I hope we will remove all polyfills after we agree on supported browsers.

Reply:
Alright, it seemed to me from [these examples](https://www.npmjs.com/package/intersection-observer-polyfill) that we should be able to use `import`.

Reply:
_should_ - we tried that 😉 , just removed the polyfill  🎉